components/kyma-environment-broker/internal/broker/broker.go

```diff
@@ -146,6 +132,19 @@ func (b *KymaEnvBroker) Provision(ctx context.Context, instanceID string, detail
 		SecretName: b.Config.SecretName,
 	}
 
+	err = b.Storage.Instances().Insert(internal.Instance{
+		InstanceID:             instanceID,
+		RuntimeID:              "",
```

Comment:
In table schema, this value cannot be null

Reply:
discussed, I will set the `runtimeID` after the response from the provisioner